### PR TITLE
Reverted copier config to be compatible with copier v5

### PR DESCRIPTION
**Pull request by paszynski**

### CODE REVIEW

File: copier.yml [hunk 1]
```diff
@@ -38,41 +38,36 @@ use_ingestion:
 
 ingestion_framework:
     type: str
-    when: "{% if use_ingestion %}true{% endif %}"
     help: What ingestion framework would you like to use?
     choices:
         Airbyte: airbyte
     default: airbyte
```
Comment (p-pekala) [inline copier.yml:44]: Let's remove this. We will have only one framework for know

File: copier.yml [hunk 2]
```diff
@@ -38,41 +38,36 @@ use_ingestion:
...
 source_id_dev:
     type: str
-    when: "{% if use_ingestion and ingestion_framework == 'airbyte' %}true{% endif %}"
     help: Source Id for dev instance
 
 workspace_id_dev:
     type: str
-    when: "{% if use_ingestion and ingestion_framework == 'airbyte' %}true{% endif %}"
     help: Workspace id for dev instance
+    default: a7e79db0-7f4f-4663-94db-ba392860fa90
```
Comment (p-pekala) [inline copier.yml:57]: I was thinking about putting it into the template and removing questions. What do you think?

File: copier.yml [hunk 3]
```diff
@@ -38,41 +38,36 @@ use_ingestion:
...
 source_id_prod:
     type: str
-    when: "{% if use_ingestion and ingestion_framework == 'airbyte' %}true{% endif %}"
     help: Source Id for prod instance
 
 workspace_id_prod:
     type: str
-    when: "{% if use_ingestion and ingestion_framework == 'airbyte' %}true{% endif %}"
     help: Workspace id for prod instance
+    default: a7e79db0-7f4f-4663-94db-ba392860fa90
```
Comment (p-pekala) [inline copier.yml:70]: same as for dev

**Review by p-pekala — CHANGES_REQUESTED** [review]

File: copier.yml [hunk 4]
```diff
@@ -38,41 +38,36 @@ use_ingestion:
...
 workspace_id_dev:
     type: str
-    when: "{% if use_ingestion and ingestion_framework == 'airbyte' %}true{% endif %}"
     help: Workspace id for dev instance
+    default: a7e79db0-7f4f-4663-94db-ba392860fa90
 
 destination_id_prod:
     type: str
-    when: "{% if use_ingestion and ingestion_framework == 'airbyte' %}true{% endif %}"
     help: Destination Id for prod instance
```
Comment (p-pekala) [inline copier.yml:61]: Maybe there could be and default as None? then if someone doesn't need it just click enter.

**Review by p-pekala — CHANGES_REQUESTED** [review]

File: copier.yml (hunk 1 above)
Comment (paszynski) [reply copier.yml:44]: Even with default? So we don't want to show future possibilities? I guess that's fine, we don't need to bother users when there is no choice anyway.

File: copier.yml (hunk 4 above)
Comment (paszynski) [reply copier.yml:61]: Should this apply to all prod ingestion questions?

File: copier.yml (hunk 2 above)
Comment (paszynski) [reply copier.yml:57]: Yeah, we should prune some variables for now, questions will seem more relevant. I was thinking, maybe I could remove `use_ingestion` for now?

**Review by p-pekala — APPROVED** [review]

**Pull request merged**